### DISCUSSION TITLE
Unable to add collaborator to my repository

### DISCUSSION THREAD

**Discussion by wencheng2** [community/community#19330] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm trying to add a collaborator to my personal repository (followed the steps on this [documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository)).

However I get stuck at step 7 - **Add name to repository**. 
To be clear, I searched for a user > clicked their name and I get this instead:

<img width="227" alt="Screen Shot 2022-06-24 at 1 23 21 pm" src="https://user-images.githubusercontent.com/103392823/175457001-702206a7-5488-4ca5-a3b9-3dc305754a9c.png">
From the image above, the button never changes to "Add name to repository".

Is anyone able to help me solve this? 
First time posting, so please let me know if I need to provide more info.

**Comment by melvincwng** [comment]
@wencheng2, do you mind providing the username of the user which you are trying to add?

**Reply by wencheng2** [reply]
it's @SteelSahaquiel but it doesn't matter what username/email address I used. 
Tried clearing cookies, using different browsers and different computers. Nothing worked.

**Reply by SteelSahaquiel** [reply]
I am not sure what happens here maybe is something on my setup.

**Reply by wencheng2** [reply]
Maybe it's my account/repository...

**Comment by melvincwng** [comment]
@wencheng2, I have tested it using a random username on my end, and was able to invite that user to be a collaborator in one of my repository.

Not sure whether you are using mobile or desktop. When I tested on mobile, it was working. You might want to retry this again on your mobile devices.

**Reply by melvincwng** [reply]
@wencheng2 let me know if this is helpful

**Reply by wencheng2** [reply]
I was using the browser version of GitHub. 
Where do I find the link to add collaborators on mobile?

This is my repository but I can't find an option to open its settings on mobile:
![Screenshot_20220624-155418_GitHub](https://user-images.githubusercontent.com/103392823/175475120-3fbfc92c-abff-4954-844d-e62445897069.jpg)

**Comment by melvincwng** [comment, marked as answer]
@wencheng2, I tested it with that specific username you've provided... Indeed there is an issue with adding that specific user.

My suggestion would be to raise a ticket to Github Support over [here](https://support.github.com/contact/account) and let them investigate this issue.

Hope this helps.

**Reply by melvincwng** [reply]
If this reply has helped you, please kindly 'Mark as answer'. Thanks!

**Reply by wencheng2** [reply]
Strange, just tried and I can't even add you. Good point about raising a ticket, will do that now.
Thank you for taking the time to help me!

**Reply by melvincwng** [reply]
@wencheng2, unfortunately it seems to me that its probably an issue on Github's end. 

The support engineers would have to check it out, after you raised a ticket.

Other than that, there's nothing much we can do as well.

No worries, take care and thanks for informing the community about this potential issue of being unable to add certain users as collaborators 😄

**Reply by wencheng2** [reply]
Just following up on this answer to say it was a bug and that GitHub support has fixed it.
Thanks again for your help!

**Reply by melvincwng** [reply]
@wencheng2, awesome! Thanks for the update, glad to know this issue has been resolved 😄
